plugins/modules/provision_workflow_manager.py

```diff
@@ -74,28 +74,41 @@
             type: list
             elements: str
         dynamic_interfaces:
-            description: Interface details of the controller
+            description: Interface details of the wireless controller
             type: list
             elements: dict
             suboptions:
+                interface_name:
+                    description: The name of the interface
+                    type: str
+                vlan_id:
+                    description: The VLAN ID associated with the interface.
+                    type: str
                 interface_ip_address:
-                    description: Ip Address allocated to the interface
+                    description: The IP address assigned to the interface.
                     type: str
-                interface_netmask_in_c_i_d_r:
-                    description: Ip Address allocated to the interface
-                    type: int
                 interface_gateway:
-                    description: Ip Address allocated to the interface
+                    description: The gateway IP address for the interface.
                     type: str
                 lag_or_port_number:
-                    description: Ip Address allocated to the interface
-                    type: int
-                vlan_id:
-                    description: Ip Address allocated to the interface
-                    type: int
-                interface_name:
-                    description: Ip Address allocated to the interface
+                    description: The port number or LAG (Link Aggregation Group) identifier.
                     type: str
+                interface_netmask_in_c_i_d_r:
```

Comment:
CAn you move this after interface_ip_address?

interface_name
vlan_id
ip
netmask
gateway
lag_or_portnumber